rest/shared/src/test/scala/io/udash/rest/HasFakeInstances.scala

```diff
@@ -0,0 +1,7 @@
+package io.udash.rest
+
+trait HasFakeInstances
```

Comment:
Because there is no more implicit full automatic macro materialization of `AsRawRPC`/`AsRealRPC`. These instances must be declared manually or through companion object, like `GenCodec`.